physical/postgresql/postgresql_test.go

```diff
@@ -260,13 +260,23 @@ func testPostgresSQLLockRenewal(t *testing.T, ha physical.HABackend) {
 	// Cancel attempt after lock ttl + 1s so as not to block unit tests forever
 	stopCh := make(chan struct{})
 	timeout := time.Duration(lock.ttlSeconds)*time.Second + lock.retryInterval + time.Second
-	time.AfterFunc(timeout, func() {
+
+	var leaderCh2 <-chan struct{}
+	newlockch := make(chan struct{})
```

Comment:
@catsby I just want to say thanks for sharing this PR with me, because you just showed me something that will be helpful for a problem that I'm working on.

Basically, I was always being lazy and using `make` to create channels, but this ["initializes an object"](https://golang.org/pkg/builtin/#make), therefore, `make`ing a `chan` returns not-nil

https://play.golang.org/p/zjUdHg2gNDN

Reply:
Hey @mccurdyc - happy to be of some help! I admit though I'm not sure how I really helped here, or if you are asking a question? The non-nil chan is used in the goroutine to capture the value asynchronously, but still have use of the value outside of the routine. In hindsight maybe that was a backwards way of doing it 🤔

Reply:
@catsby no questions. You helped by reminding me that it if you want an un-initialized channel for checking `nil` while it is still un-initialized, you don't use `make`, instead use `var`.